### ISSUE TITLE
DDC (direct data capture) support

### ISSUE THREAD

**Issue by rkrenn** [phoenixctms/ctsms#253]
after long discussions, we try to avoid implementin a (android/iOS) app for DDC.

instead, we will introduce a "Synchronize" service or job, which can be activated for a trial (one shot or periodic etc.).

this job is supposed to synchronize eCRF data between 2 Phoenix CTMS instances via Rest-API.

For DDC, the mobile device will then run a local Phoenix instance, which will synchronize with the central one.

**Comment by josharmenta** [comment]
@rkrenn We're implementing this using Apple's ResearchKit, it's working well so far.

**Comment by rkrenn** [comment]
so you implemented a dedicated iOS app using ResearchKit framework.
does it use a synchronization implementation toward the Phoenix Rest-API, like outlined in https://github.com/carekit-apple/CareKit/issues/322 ?

sounds like quite some work!

**Comment by josharmenta** [comment]
Yes I got it working the other day. I need to refactor it majorly to sanitize it of anything specific to our trials, but once that happens I'll put it up on our GH page, along with a few other things that we've set up for Phoenix. Basically I wrote a library that wraps the API calls and did a circular buffer from the ResearchKit survey module.

**Comment by josharmenta** [comment]
In the future we're planning to pull data from the watches too and upload those into Phoenix, when we get there, I'll let you know and post that repo as well.
